### ISSUE TITLE
Decision document ID – copy function not accessible to AT

### ISSUE THREAD

**Issue by alisan16** [department-of-veterans-affairs/caseflow#12098]
The role of the Decision Document ID cannot be determined with assistive technology programs. Visually there is an icon indicating this is a copy function as well as a mouse over that reads: Click to Copy. (New: 08/22/2019)
